src/components/StylisedIcon/index.tsx

```diff
@@ -0,0 +1 @@
+export { StylisedIcon } from './StylisedIcon'
```

Comment:
Sounds better to me tbh, WDYT @Marruk?

Reply:
I like it too!

If it's ok with you I would like to merge it like this because it's called `StylisedIcon` in both Figma and the app design systems. I'll put it on my list to rename it everywhere in one go and keep it the same.